package/README.md

```diff
@@ -1,44 +1,81 @@
-# GOV.UK Frontend - All components
+# GOV.UK Frontend
 
-All of the components in GOV.UK Frontend, in a single package.
+GOV.UK Frontend contains the code you need to start building a user interface
+for government platforms and services.
 
-## Guidance
+See live examples of GOV.UK Frontend components, and guidance on when to use
+them in your service, in the [GOV.UK Design
+System](https://govuk-design-system-production.cloudapps.digital/).
 
-Find out when to use the Back link component in your service in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital).
+## Contact the team
 
-## Dependencies
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
+team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
 
-To consume all components you must be running npm version 5 or above.
+## Requirements
 
-## Installation
+To use GOV.UK Frontend with [npm](https://www.npmjs.com/) you must:
 
-    npm install --save govuk-frontend
+Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM. The minimum version of Node required is 8.6.0.
```

Comment:
Is this true?

Reply:
No, this is the minimum we require to contribute GOV.UK Frontend, not to consume it.

Reply:
I've removed this section as it duplicates the [npm installation instructions](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md) and is not true if you decide to install with option 2. 

In the npm installation instructions, I've downgraded the version of node required to 4.2.0. which builds with nunjucks 3.0.0 and the nunjucks templates render.

Our [contributing docs](https://github.com/alphagov/govuk-frontend/blob/master/CONTRIBUTING.md) still say that Node 8.6.0 is required.